include/pstl_offload/internal/usm_memory_replacement.h

```diff
@@ -289,53 +351,67 @@ inline void* __attribute__((always_inline)) __libc_realloc(void *__ptr, std::siz
 inline void* __attribute__((always_inline))
 operator new(std::size_t __size)
 {
-    return ::__pstl_offload::__internal_operator_new(__size, alignof(std::max_align_t));
+    return ::__pstl_offload::__internal_operator_new(__size, 0);
```

Comment:
Should we use `__not_use_explicit_alignment` here and in other otherloads?

Reply:
Sure, missed that.